File: 0d_files/python/simple.py
import sys


if __name__ == "__main__":


    with open(sys.argv[1]) as fh:
        count = 0


        for line in fh:
            elems = line.split(',')
            if elems[1] == "Tanzania":
                count += 1



    print(f"found {count} sales for Tanzania")

File: 0d_files/rust/simple.rs
use std::fs::File;
use std::io::{self, BufRead, BufReader};

fn main() -> io::Result<()> {
    let args: Vec<String> = std::env::args().collect();

    let file = File::open(&args[1])?;
    let reader = BufReader::new(file);

    let mut found = 0usize;
    for line in reader.lines() {
        let line = line?;
        let elems: Vec<&str> = line.split(',').collect();
        if elems[1] == "Tanzania" {
            found += 1;
        }
    }
    println!("found {} sales for Tanzania", found);
    Ok(())
}
